src/app/modules.ts

```diff
@@ -53,7 +54,8 @@ export const CUSTOM_MODULES = [
     IFrameModule,
     ListingDataTableModule,
     SearchableDropDownModule,
-    SiteSelectorModule
+    SiteSelectorModule,
+    DotDialogMessageModule
```

Comment:
I don't know if this is a good name, if tomorrow we change the dialog for something else then the name is not appropriate anymore, so it shoud be something like: DotLargeMessageModule

same for DotDialogMessageComponent and DotDialogMessageService